pax-macro/src/lib.rs

```diff
@@ -300,7 +300,7 @@ pub fn pax_file(args: proc_macro::TokenStream, input: proc_macro::TokenStream) -
     let mut content = String::new();
     let _ = file.unwrap().read_to_string(&mut content);
     let stream: proc_macro::TokenStream = content.parse().unwrap();
-    pax_internal(stream, input, true, Some(include_fix))
+    pax_internal(stream, input, false, Some(include_fix))
```

Comment:
note: this hard-coded boolean logic for `is_root` inside `pax_file` is incorrect.  It only happened to work before because we were testing with a single root component.  Now it will only happen to work so long as `pax_file` is used for _non_-application root components.  A more robust fix will be to expose a separate API for specifying the application-root component, so that we robustly handle the cross product `{root, not_root} ⨯ {inline, file}`